Good-Bids/goodbids
# [#217] Setting up the basic custom email file structure

# Summary
Customizing WooCommerc email notifications is new to me so I read up on a few articles and this is the [one that helped me the most](https://www.godaddy.com/resources/skills/how-to-add-a-custom-woocommerce-email).  

This is a basic setup of the email notification files structure. It sets up the email `Auction Live for Watchers` and loads it into the `woocommerce_email_classes`. 

This PR does nothing else but set up the file structure and load the custom email notification. 

## Issues

* #217

## Testing Instructions

1. Go to WooCommerce -> Settings -> Emails
2. Make sure you have the custom email  `Auction Live Watchers`

## Screenshots

NA
